main.nf

```diff
@@ -268,114 +200,382 @@ ${summary.collect { k,v -> "            <dt>$k</dt><dd><samp>${v ?: '<span style
    return yaml_file
 }
 
-
->>>>>>> TEMPLATE
 /*
  * Parse software version numbers
  */
 process get_software_versions {
-<<<<<<< HEAD
-    validExitStatus 0,2
-=======
->>>>>>> TEMPLATE
+    validExitStatus 0
 
     output:
     file 'software_versions_mqc.yaml' into software_versions_yaml
 
     script:
     """
-<<<<<<< HEAD
-    echo $params.version > v_pipeline.txt
+    echo $params.pipelineVersion > v_pipeline.txt
```

Comment:
Might want to use `manifest.version` for this purpose. Just define these in the `nextflow.config` and then use the variables set there for these kind of calls. Works also for nextflow version etc pp.... 

https://github.com/nf-core/tools/blob/54e233a4c167f515b6f616b3c5a8a9bd660861c0/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/nextflow.config#L85